tools/ovc/openvino/tools/ovc/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -35,7 +35,7 @@ def get_pytorch_decoder(model, example_inputs, args):
     inputs = prepare_torch_inputs(example_inputs)
     if not isinstance(model, (TorchScriptPythonDecoder, TorchFXPythonDecoder)):
         if isinstance(model, torch.export.ExportedProgram):
-            raise RuntimeException("Models recieved from torch.export are not yet supported by convert_model.")
+            raise RuntimeError("Models recieved from torch.export are not yet supported by convert_model.")
```

Comment:
nit: please fix it in the next PR:

```suggestion
            raise RuntimeError("Models received from torch.export are not yet supported by convert_model.")
```